src/tasks/AotCompilerTask/MonoAOTCompiler.props

```diff
@@ -15,25 +13,31 @@
   <ItemGroup Condition="'$(TargetOS)' == 'ios' or '$(TargetOS)' == 'tvos' or '$(TargetOS)' == 'maccatalyst'">
     <MonoAOTCompilerDefaultAotArguments Include="direct-icalls" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetOS)' == 'android'">
-    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi" />
-    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm64'" Include="mtriple=aarch64-linux-android" />
-    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'x86'" Include="mtriple=i686-linux-android" />
-    <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'x64'" Include="mtriple=x86_64-linux-android" />
-  </ItemGroup>
   <ItemGroup Condition="'$(TargetOS)' == 'android' or '$(TargetOS)' == 'ios' or '$(TargetOS)' == 'iossimulator' or '$(TargetOS)' == 'tvos' or '$(TargetOS)' == 'tvossimulator' or '$(TargetOS)' == 'maccatalyst'">
     <!-- Default trampolines run out for libraries tests -->
     <MonoAOTCompilerDefaultAotArguments Include="nimt-trampolines=2000" />
     <MonoAOTCompilerDefaultAotArguments Include="ntrampolines=10000" />
     <MonoAOTCompilerDefaultAotArguments Include="nrgctx-fetch-trampolines=256" />
     <MonoAOTCompilerDefaultAotArguments Include="ngsharedvt-trampolines=4400" />
     <MonoAOTCompilerDefaultAotArguments Include="nftnptr-arg-trampolines=4000" />
-    <MonoAOTCompilerDefaultAotArguments Include="nrgctx-trampolines=21000" />
```

Comment:
Interesting, seems we had different values between sample and android/build. Great to see it unified